scripts/build_components.sh

```diff
@@ -75,7 +75,7 @@ for dir in "${components_to_build[@]}"; do
 
   echo "Updating the image version in the fondant_component.yaml with:"
   echo "${full_image_names[0]}"
-  sed -i "s|^image: .*|image: ${full_image_names[0]}|" fondant_component.yaml
+  sed -i '' "s|^image: .*|image: ${full_image_names[0]}|" fondant_component.yaml
```

Comment:
This doesn't seem to work on linux.

Reply:
The platform independent command seems to be `sed -i -e` ([link](https://unix.stackexchange.com/questions/13711/differences-between-sed-on-mac-osx-and-other-standard-sed))

Reply:
This seems to work, however it also creates a yaml-e file besides yaml. So will add yaml-e to gitignore